gateway-api-spec.yaml

```diff
@@ -880,7 +880,7 @@ components:
             amount:
               token_identifier:
                 rri: xrd_rr1qy5wfsfh
-              value: '123000000000000000'
+              value: '100'
```

Comment:
Just a thought, but I'd rather a larger number here, to be indicative of a quantity someone might send. If they were rushing and didn't realise they were subunits, the large number might hint to them that they need to read closer!

Reply:
sure